src/app/app.component.ts

```diff
@@ -241,19 +251,16 @@ export class AppComponent {
   }
 
   private hideZendeskChat() {
-    let zendeskElements = this._dom.bySelectorAll('.zopim')
-
-    if (zendeskElements && zendeskElements.length > 1) {
-      zendeskElements[0]['style']['display'] = 'none' // ChatWidgetButton
-      zendeskElements[1]['style']['display'] = 'none' // ChatWidgetWindow
+    let zendDeskElement = this._dom.byId('launcher')
+    if (zendDeskElement) {
+      zendDeskElement.style.display = 'none';
     }
   }
 
   private showZendeskChat() {
-    let zendeskElements = this._dom.bySelectorAll('.zopim')
-
-    if (zendeskElements && zendeskElements.length > 1) {
-      zendeskElements[0]['style']['display'] = 'block' // ChatWidgetButton
+    let zendDeskElement = this._dom.byId('launcher')
+    if (zendDeskElement) {
+      zendDeskElement.style.display = null;
```

Comment:
The display null looks a little unusual, although it seems to work. Would `initial` be a more standard value for display?

Reply:
A colleague of mine - who is a bit of a js expert - recommended this approach. It will reset itself to the original value of the dom. Sort if like a reset. 

I'll investigate the use of `initial` since the intent is much clearer

Reply:
Tested with `initial` and it works the same so I've changed it from null to `initial`